### PR TITLE
add maven_01 task

### PR DESCRIPTION
**Pull request by wilden5**
Added the maven_01 task in which I created the multi-module maven project in Idea

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>JavaAQA</groupId>
+    <artifactId>OnlineStore</artifactId>
+    <packaging>pom</packaging>
+    <version>1.0-SNAPSHOT</version>
+    <modules>
+        <module>parent</module>
```
Comment (artemmoroz) [inline pom.xml]: parent is not a module, it is parent (in your case this pom should be a parent)

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>JavaAQA</groupId>
```
Comment (artemmoroz) [inline pom.xml]: naming conventions

**Review by artemmoroz — CHANGES_REQUESTED** [review]
Please
1. correct naming, according to maven guides (artifactid and so on)
2. create a proper gitignore, https://www.toptal.com/developers/gitignore
3. correct modules structure

**Comment by wilden5** [comment]
> Please
> 
> 1. correct naming, according to maven guides (artifactid and so on)
> 2. create a proper gitignore, https://www.toptal.com/developers/gitignore
> 3. correct modules structure

Hello, I changed the project structure and maven naming in the project. Also, edited gitignore file.

**Review by artemmoroz — CHANGES_REQUESTED** [review]
according prev comments everything is fine, but you ve already commited files that should be ignored *idea, *iml and so on, you should remove them https://stackoverflow.com/questions/19663093/apply-gitignore-on-an-existing-repository-already-tracking-large-number-of-file

**Comment by wilden5** [comment]
> according prev comments everything is fine, but you ve already commited files that should be ignored *idea, *iml and so on, you should remove them https://stackoverflow.com/questions/19663093/apply-gitignore-on-an-existing-repository-already-tracking-large-number-of-file

Removed unnecessary files from the project. Thanks for helping me find the right git console commands!

**Review by artemmoroz — APPROVED** [review]

**Pull request merged**